libs/utils/analysis/latency_analysis.py

```diff
@@ -331,6 +344,123 @@ def plotLatencyBands(self, task, axes=None):
                 axes.axvspan(start, end, facecolor='b', alpha=0.1)
         except: pass
 
+    def plotActivations(self, task, tag=None, threshold_ms=16):
+        """
+        Plots "activation intervals" for the specified task
+
+        An "activation interval" is time incurring between two consecutive
+        wakeups of a task. A set of plots is generated to report:
+        - Activations interval at wakeup time: every time a task wakeups a
+          point is plotted to represent the time interval since the previous
+          wakeup.
+        - Activations interval cumulative function: reports the cumulative
+          function of the activation intervals.
+        - Activations intervals histogram: reports a 64 bins histogram of
+          the activation internals.
+
+        All plots are parameterized based on the value of threshold_ms, which
+        can be used to filter activations intervals bigger than 2 times this
+        value.
```

Comment:
OK... still seems strange to me but doesn't add any complexity so no problem I guess.